test/unit/config/matchers/to-be-positioned-popover.js

```diff
@@ -5,10 +5,19 @@
  * @param {HTMLElement} element Popover element.
  */
 function toBePositionedPopover( element ) {
-	const pass = element.classList.contains( 'is-positioned' );
+	const popover = element && element.closest( '.components-popover' );
+	const isPopoverPositioned =
+		popover && popover.classList.contains( 'is-positioned' );
+	const pass = !! popover && !! isPopoverPositioned;
```

Comment:
Some style nits:
- can use `element?.closest` and `popover?.classList` optional chainings
- `pass` can be simply `!! isPopoverPositioned`, the `!! popover` condition is redundant.